style/buttons.less

```diff
@@ -0,0 +1,20 @@
+@btn-border-radius: 4px;
+@primary-background: #f47642;
+@primary-hover-background: #fa804d;
+
+// Override the default text color from bootstrap-material-design
+.btn-primary,
+.btn-success { color: @white !important; }
+
+.btn-primary {
+	font-size: 18px;
```

Comment:
Looks like dreamweaver uses tabs instead of spaces. Can you change these to 2-spaces?